src/components/BookmarkButton.js

```diff
@@ -3,7 +3,7 @@ import styled from 'styled-components';
 export default function BookmarkButton({ onClick, name, isBooked }) {
   return (
     <Button onClick={onClick} isBooked={isBooked}>
-      {(name = 'Save this plant')}
+      {isBooked ? 'Saved Plant' : 'Click to save this plant'}
     </Button>
   );
 }
```

Comment:
For better readability I would recommend to write `isBookmared` and not a short version `isBooked`. But that is just my personal opinion 🙂